containersQa/testlib.bash

```diff
@@ -570,6 +570,13 @@ function s2iBasic() {
   local d=`mktemp -d`
   pushd $d
     mvn -B archetype:generate -DgroupId=org.test.rhimg  -DartifactId=rhimg -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4 -DinteractiveMode=false
+    # Need to modify this generated pom file for a maven.compiler.source & target to be 1.8 rather than 1.7.
+    pushd rhimg
+       sed -i "s;<maven.compiler.source>1.7</maven.compiler.source>;<maven.compiler.source>1.8</maven.compiler.source>;g" pom.xml
+       sed -i "s;<maven.compiler.target>1.7</maven.compiler.target>;<maven.compiler.target>1.8</maven.compiler.target>;g" pom.xml
```

Comment:
same here, the  -DarchetypeVersion=1.4  bumped to neer should be all, with small detail, that there is no new archetype of maven-archetype-quickstart ... SO the way should be to update it. But I can guess why you are ot up to it. Maybe @mkoncek knows?

Reply:
Same as above. 1.4 is the latest

Reply:
I barely remember us having problems with archetype...

Reply:
hi! There is better way then that sed:
https://github.com/apache/maven-archetypes/blob/d605cb7cb756618db9ee22af2880c6d7bdde2537/maven-archetype-quickstart/src/main/resources-filtered/archetype-resources/pom.xml#L48

You should be able to pass -DjavaCompilerVersion=8  and be doen with that.

Reply:
hm..jsut tried and 
```
mvn -B archetype:generate -DgroupId=org.test.rhimg  -DartifactId=rhimg -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4 -DinteractiveMode=false  -DjavaCompilerVersion=11
```

and it did NOT did the job

Reply:
attempted:
https://github.com/apache/maven-archetypes/pull/18/files#diff-24fd1babb342805ebbb676979bb49bb4011b4fe28c396dc9b6e962fe2ac54c10R49

filled:
https://issues.apache.org/jira/projects/MARCHETYPES/issues/MARCHETYPES-76
https://issues.apache.org/jira/projects/MARCHETYPES/issues/MARCHETYPES-77